style.css

```diff
@@ -0,0 +1,73 @@
+.header {
+  background-color: #3c3a39;
+  height: 48px;
+  width: 375px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header p {
+  color: #fff;
+  font-family: 'Roboto', sans-serif;
+  margin-left: 24px;
+}
+
+.header img {
+  height: 24px;
+  width: 24px;
+  margin-right: 24px;
+}
+
+.mainContain {
+  background-color: #1c1a19;
+  height: 590px;
+  width: 375px;
+  top: 92px;
+  display: flex;
+  flex-direction: column;
+  background-image: url("images/Illustration.png");
+  background-repeat: no-repeat;
+  background-position: right bottom -45px;
+}
+
+.informativeText {
+  flex-direction: column;
+  margin-left: 24px;
+}
+
+.informativeText h1 {
+  display: flex;
```

Comment:
- [ ] This is great using flexbox to layout your content, congrats 👏 , But we can still improve this code by using DRY (don't repeat yourself) but why? well, I realized that you have used the property `display: flex` at multiple places, please let's keep the code clean by extracting the property into a separate class that can be reused at multiple places, see the example below: 

- CSS file
```
.d-flex {
  display: flex;
}

.flex-center {
  align-items: center;
  justify-content: center;
}
```
- HTML file
```
<div class="d-flex flex-center">{your content}</div>
```

And remember you can do this for all the other properties that seem to be repeated multiple times, With this, I promise you'll be coding like a pro 😉

Reply:
This was the most challenging for me, but now I understand better how to improve the code.
Thanks a lot, Baraka!! 😄  ✅